.expeditor/build.habitat.yml

```diff
@@ -0,0 +1,3 @@
+---
+origin: scotthain
+smart_build: true
```

Comment:
What makes it "smart"?

Reply:
Expeditor only rebuilds packages that have been modified if smart build is true. (This actually will have no effect currently because all this PR does is *create* the pipeline